wgsl/index.bs

```diff
@@ -2990,7 +2990,7 @@ TODO: deduplicate these tables
 `vec4<type> textureLoad(texture_ro, i32 coords, i32 level_of_detail)`
 `vec4<type> textureLoad(texture_ro, vec2<i32> coords, i32 level_of_detail)`
 `vec4<type> textureLoad(texture_ro, vec3<i32> coords, i32 level_of_detail)`
-  %32 = OpImageLoad %v4float %texture_ro %coords Lod %level_of_detail
+  %32 = OpImageRead %v4float %texture_ro %coords Lod %level_of_detail
```

Comment:
Right. There is OpImageLoad.  !

This should be OpImageFetch.

OpImageFetch is used with images that are set up to have a sampler (OpTypeImage Sampled=1), but not actually use the sampler.
OpImageRead is used with storage images (OpTypeImage Sampled=2). (Note that Sampled=0 is not permitted by Vulkan)

Reply:
Whoops.  I was reading the wrong section.
You're right, *this* one is OpImageRead, and should not take a level-of-detail operand.